backend/pkg/httpserver/list_aggregated_feature_support.go

```diff
@@ -35,9 +52,24 @@ func (s *Server) ListAggregatedFeatureSupport(
 		return cachedResponse, nil
 	}
 
+	var targetMobileBrowser backend.BrowserPathParam
+	if request.Params.IncludeBaselineMobileBrowsers != nil {
+		var err error
+		targetMobileBrowser, err = getDesktopsMobileProduct(request.Browser)
+		if err != nil {
+			return backend.ListAggregatedFeatureSupport400JSONResponse{
+				Code:    400,
+				Message: err.Error(),
+			}, err
+		}
+	} else {
+		targetMobileBrowser = request.Browser
+	}
```

Comment:
I don't know if this will fly (the mobile browser being set to the same as the target browser for requests that aren't looking for combined data), but the functionality should be the same.